guides/source/upgrading_ruby_on_rails.md

```diff
@@ -276,6 +276,22 @@ class Notifier < ActionMailer::Base
 end
 ```
 
+### Foreign Key support
+
+Rails 4.2 adds basic support for foreign keys. If the
+[Foreigner gem](https://github.com/matthuhiggins/foreigner) were been used to
+get support for foreign keys in the app, you can consider removing the gem
+from Gemfile on upgrading to 4.2 and using the default support.
+
+But remember that some optional and adapter specific options can not work as
```

Comment:
But remember that some optional and adapter specific options **might** not work as expected